src/Mutator/Extensions/BCMath.php

```diff
@@ -106,46 +107,71 @@ public function canMutate(Node $node): bool
         return isset($this->converters[$node->name->toLowerString()]);
     }
 
-    private function setupConverters(array $functionsMap): void
+    /**
+     * @return array<string, Closure>
+     */
+    private static function createConverters(array $functionsMap): array
     {
-        $converters = [
-            'bcadd' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Plus::class)
-            )),
-            'bcdiv' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Div::class)
-            )),
-            'bcmod' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Mod::class)
-            )),
-            'bcmul' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Mul::class)
-            )),
-            'bcpow' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Pow::class)
-            )),
-            'bcsub' => $this->makeCheckingMinArgsMapper(2, $this->makeCastToStringMapper(
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Minus::class)
-            )),
-            'bcsqrt' => $this->makeCheckingMinArgsMapper(1, $this->makeCastToStringMapper(
-                $this->makeSquareRootsMapper()
-            )),
-            'bcpowmod' => $this->makeCheckingMinArgsMapper(3, $this->makeCastToStringMapper(
-                $this->makePowerModuloMapper()
-            )),
-            'bccomp' => $this->makeCheckingMinArgsMapper(2,
-                $this->makeBinaryOperatorMapper(Node\Expr\BinaryOp\Spaceship::class)
-            ),
-        ];
-
-        $functionsToRemove = array_filter($functionsMap, static function (bool $isOn): bool {
-            return !$isOn;
-        });
-
-        $this->converters = array_diff_key($converters, $functionsToRemove);
+        return array_intersect_key(
```

Comment:
hm that CS change is the fixer I think... well at least it doesn't hit the 140 limit now I guess